### PR TITLE
Add a simplified script environment constructor for emulator

### PR DESCRIPTION
**Pull request by pattyshack**

### CODE REVIEW

**Review by turbolent — APPROVED** [review]
Nice, thank you!

**Review by janezpodhostnik — APPROVED** [review]

File: fvm/environment/facade_env.go [hunk 1]
```diff
@@ -157,6 +158,33 @@ func NewScriptEnvironment(
 		})
 }
 
+// This is mainly used by the emulator and cadence tools testing.
+func NewEmulatorScriptEnvironment(
```
Comment (pattyshack) [inline fvm/environment/facade_env.go:162]: looks like we have some command line tool in flow-go which could also switch to this.  I'll rename this function and update the cmd line tool while i'm at it.

Comment (pattyshack) [reply fvm/environment/facade_env.go:162]: hmm.  looks like cmd line change is a bit involved.  I'll only update the function name in this pr.

**Comment by pattyshack** [comment]
bors merge

**Comment by pattyshack** [comment]
bors merge

**Pull request merged**